virt_lightning/__init__.py

```diff
@@ -159,25 +159,31 @@ def start(self, domain):
         domain.attachDisk(meta_data_iso, device="cdrom", disk_type="raw")
         domain.dom.create()
 
+    def find_kvm_binary(self):
+        paths = ["/usr/bin/qemu-kvm", "/usr/bin/kvm"]
```

Comment:
This will cover all paths:
```
paths = os.environ['PATH'].split(':')
```

Reply:
I tried to keep it dumb for now. I believe this is enough to cover all the major Linux distro. This being said, I will be happy to apply a patch if someone come with a better code.

Is this the current method a blocker for you?

Reply:
Nope, I see ENV variable and request rewrite that. It might enough but i'm not sure. I can write some code over kvm and define path to binaries to my own build and your solution won't work in that case. Anyway if someone request that we can improve that part of code.

Reply:
:+1: